.remarkrc.js

```diff
@@ -0,0 +1,43 @@
+exports.plugins = [
+  [
+    "@double-great/remark-lint-alt-text",
```

Comment:
If you'd like this linter to fail (rather than warn), you can set the severity like this:
```suggestion
    ["@double-great/remark-lint-alt-text", [2]],
```